tests/conftest.py

```diff
@@ -77,10 +78,20 @@ def setup_hotfix_check(request, ansible_module):
         assert result['rc'] == 0
 
     def teardown_hotfix_check():
+        teardown = ansible_module.command(Packages.unlock())
```

Comment:
@jameerpathan111 this will be applicable only for 6.6 and above. right? If yes, can we make it to run only for >= 6.6

Reply:
@ntkathole updated the PR.